osu.Game.Tests/Visual/Gameplay/TestSceneReplayPlayer.cs

```diff
@@ -23,14 +24,29 @@ public override void SetUpSteps()
             AddUntilStep("player loaded", () => Player.IsLoaded);
         }
 
-        [Test]
-        public void TestPause()
+        [TestCase("space")]
+        [TestCase("mouse middle")]
+        public void TestPause(string action)
```

Comment:
Just make two test scenes at this point. Using a string like this is a bit weird.